test/client/constructor.ts

```diff
@@ -4,7 +4,7 @@ import { Client } from "xrpl-local";
 
 describe("client constructor", function () {
   it("Client - implicit server port", function () {
-    new Client("wss://s1.ripple.com");
```

Comment:
You can also just ignore the rule here, because you're specifically testing the constructor. I'd do that instead, because it makes the code more readable.